src/sentry/static/sentry/app/components/forms/selectField.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import FormField from './formField';
+import StyledSelect from './select.styled';
+
+export default class SelectField extends FormField {
+  static propTypes = {
+    ...FormField.propTypes,
+    options: PropTypes.array.isRequired,
+    onChange: PropTypes.func,
+    clearable: PropTypes.bool,
+  };
+
+  static defaultProps = {
+    ...FormField.defaultProps,
+    clearable: true,
+  };
+
+  getClassName() {
+    return '';
```

Comment:
why is this defined / overridden?

Reply:
It's required by formField, but we don't want to use CSS classes in this case